samples/bot-sequential-flow-adaptive-cards/csharp/SequentialUserSpecificFlow/Manifest/manifest.json

```diff
@@ -35,6 +35,26 @@
       ]
```

Comment:
I can not see this image in Node sample, please verify once.
shareIncidentCard.png